Task 3/Insertion Sort/insertion sort_c

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
Also, please avoid spaces in the filename.

The complete filename also includes the file extension, which is `.c` in your case.

Your complete path must be something like:

`Task 3/Insertion Sort/InsertionSort.c`

Reply:
Thanks for mentioning 
Wasn't aware 
corrected.